### ISSUE TITLE
scene.move_camera() has no effect

### ISSUE THREAD

**Issue by mukral** [zauberzeug/nicegui#200]
Hi, changing the position in a scene does not seem to have an effect.
Version nicegui 1.0.4

```python
from nicegui import ui
container1 = ui.row()
container2 = ui.row()
with container1:
    cameraY = ui.number(label='cameraY', value=-100)
def drawScene(cameraY):
    container2.clear()
    with container2:
        with ui.scene(width=640, height=400) as scene:
            scene.sphere(2)
            scene.move_camera(
                cameraY-100, cameraY,50,  # position
                0, 0, 0,  # look at
                0, 0, 1,  # up
                0  # animation duration
        )            
ui.button("go", on_click=lambda: drawScene(cameraY.value))
ui.run()
```
Thank you very much for all of your efforts and especially for the major changes in 1.0!

Markus

**Comment by falkoschindler** [comment]
Hi @mukral, thanks for reporting this issue! You're right, there is something fishy with camera movements.

Since you create a new scene on every button click, it seems to be an issue with setting initial camera positions. If you would only move the camera, `move_scene` would actually do something. Here is a minimum example for moving the camera on every input value change:

```python
with ui.scene() as scene:
    scene.sphere()

ui.number(label='cameraY', value=0, on_change=lambda e: scene.move_camera(y=e.value))
```

But like in your example, the initial value 0 is not applied.

**Comment by falkoschindler** [comment]
We just released version 1.0.5 which fixes this issue.
